docs/src/main/sphinx/release/release-371.md

```diff
@@ -0,0 +1,57 @@
+# Release 371 (16 Feb 2022)
+
+## General
+
+* Remove `deprecated.disable-set-properties-security-check-for-create-ddl`
+  configuration toggle. ({issue}`10923`)
+* Improve query performance by removing redundant predicates above table scan.
```

Comment:
This behavior already existed -- the change is really about fixing a bug where the optimizer would loop forever: https://github.com/trinodb/trino/issues/10532

Reply:
Reworded and updated issue link